draftjs_exporter/composite_decorators.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import, unicode_literals
+
+import cgi
+import re
+
+
+class CompositeDecorator():
```

Comment:
In the Draft.js implementation the decorator is divided into two separate things, a `strategy` function and the `component` to render. Here they are both grouped under one class – could you explain why you made that choice?